authentik/sources/ldap/models.py

```diff
@@ -99,6 +99,11 @@ class LDAPSource(Source):
         help_text=_("Property mappings used for group creation/updating."),
     )
 
+    password_login_update_internal_password = models.BooleanField(
+        default=False,
```

Comment:
Should probably be set to `True` by default to retain the same behaviour as the current version

Reply:
See the other discussion about this. The migrations set it to true, and then back to false so that existing behavior is preserved and new sources will get false

Reply:
I think we should still keep it to true for the new sources.

Reply:
Changed back to have the default to true in the model, the migrations and the frontend.

Reply:
Could you please explain your reasoning for wanting this to be true? LDAP binds are generally fast enough do not outweigh the several pitfalls and negative effects on security. You can see my arguments above